src/App.java

```diff
@@ -1,5 +1,203 @@
+import java.util.Scanner;
+
 public class App {
+    public Random myRandom;

```

Comment:
The `Random` class is not well-named, because java already has a `Random` class built in that does something quite different from this. You should rename the class to be more clear what its for.